src/Instrumentation/Laravel/tests/Integration/LaravelInstrumentationTest.php

```diff
@@ -54,12 +20,12 @@ public function test_request_response(): void
         $response = $this->call('GET', '/');
         $this->assertEquals(200, $response->status());
         $this->assertCount(1, $this->storage);
-        $span = $this->storage->offsetGet(0);
+        $span = $this->storage[0];
```

Comment:
Psalm was complaining under PHP 8.0, after phpdoc hint was added to base TestCase.